packages/office-ui-fabric-react/src/components/Persona/examples/PersonaExample.scss

```diff
@@ -10,4 +10,7 @@
   .example-label {
     margin: 10px 0;
   }
+  .custom-example-coin img {
```

Comment:
We don't typically use the kebab-case naming convention for our classes, can you change this one and the one above to camelCase?